src/CustomPostFormat.hs

```diff
@@ -49,7 +49,7 @@ instance MimeUnrender ANewFormat String where -- This instance implements the de
     Left _ -> Left "Decoding error"
 
 handlerName :: String -> Handler String
-handlerName = return  -- Just output back the input string value
+handlerName nameIn = return nameIn -- Just output back the input string value
```

Comment:
The force push was due to me having changes the param to 'name' and not back to 'nameIn' and already having pushed.